### ISSUE TITLE
Unhandled error in lazy lookup plugin

### ISSUE THREAD

**Issue by iruyanka** [Flexget/Flexget#1097]
```
2016-04-27 00:14 ERROR    lazy_lookup   TV Shows        Unhandled error in lazy lookup plugin
Traceback (most recent call last):
  File "/usr/local/flexget/env/lib/python2.7/site-packages/flexget/utils/lazy_dict.py", line 38, in __getitem__
    func(self.store)
  File "/usr/local/flexget/env/lib/python2.7/site-packages/flexget/plugins/metainfo/tvmaze_lookup.py", line 131, in lazy_series_lookup
    series = series_lookup(**lookupargs)
  File "/usr/local/flexget/env/lib/python2.7/site-packages/flexget/utils/database.py", line 31, in wrapper
    return func(*args, **kwargs)
  File "/usr/local/flexget/env/lib/python2.7/site-packages/flexget/plugins/api_tvmaze.py", line 471, in series_lookup
    prepared_params = prepare_lookup_for_pytvmaze(**lookup_params)
  File "/usr/local/flexget/env/lib/python2.7/site-packages/flexget/plugins/api_tvmaze.py", line 429, in prepare_lookup_for_pytvmaze
    prepared_params['show_name'] = native_str(title) if title else None
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-4: ordinal not in range(128)
```

```
/usr/local/flexget/env/bin/flexget -V
2.0.4
You are on the latest release.
```

Thank you.

**Comment by cvium** [comment]
Happened to me as well after I randomly received a russian string from Trakt.tv

http://pastebin.com/h2zHaQC5

**Comment by stevezau** [comment]
should be fixed in https://github.com/Flexget/Flexget/commit/1eca4a6dd84d869b64293fabbdb4dab7fc4b39ea

**Comment by crawln45** [comment]
fixed e5ab31c
